lib/kanal/plugins/batteries_bridge/bridges/telegram_bridge.rb

```diff
@@ -66,34 +66,34 @@ def setup
             output_convert :keyboard, :tg_reply_markup do |keyboard_object, input, output|
               nil if keyboard_object.nil? || !keyboard_object.to_a.count.positive?
 
-              if output.specifics.has?(:tg_reply_keyboard) == false || output.specifics.get(:tg_reply_keyboard) == false
-                inline_keyboard = []
+              if output.specifics.get :tg_reply_keyboard
```

Comment:
Better check explicitly for true because any object or something passed here will be considered truthy. Only falsy values are false, nil.

Just in case. I doubt there will be `the case` but just to be sure we won't have any side effects